metrics/aws.rds/rds.go

```diff
@@ -194,7 +194,7 @@ func (m *RDS) Collect(ctx context.Context, levelName string) ([]blip.MetricValue
 				},
 			}
 			if isCounter[metric] {
-				m.Type = blip.COUNTER
+				m.Type = blip.DELTA_COUNTER
```

Comment:
Is this right? I'm not sure because different platforms differ on whether counter means cumulative or delta. In my mind, counter == cumulative.

Reply:
From the DD values they behaved liked delta counters. I will confirm with AWS support.

Reply:
I only did this because we currently don't have any cumulative counters for RDS. I can main convert it into a map of maps (for both delta and cumulative counters) but cumulative counters will be empty anyway, but the code will be unnecessarily more complex (I will have to add code for checking if a metric is cumulative while knowing they never are atm.

I have added a comment to make it clearer, but if you still think making it complete will be better I will be happy to do so.

Reply:
The alternative is doing something similar to:

```go
var isCounter = map[byte]map[string]bool{
	blip.DELTA_COUNTER: {
		"AbortedClients":       true,
		"BacktrackWindowAlert": true,
	},
	blip.CUMULATIVE_COUNTER: {},
}
```

And convert the check to:

```go
if isCounter[blip.DELTA_COUNTER][metric] {
	m.Type = blip.DELTA_COUNTER
}

if isCounter[blip.CUMULATIVE_COUNTER][metric] {
	m.Type = blip.CUMULATIVE_COUNTER
}
```

Reply:
Ack, comment is good; it's sufficient to help future eng avoid confusion/know what we intended to do today.